scala-native-cli-integration-test-runner/src/test/run/integration-test/test

```diff
@@ -0,0 +1,31 @@
+> compile
+
+# Print used version
+> runCli --version
+
+# -- Link and check if outpath exists
+> runCli --outpath target/out1 -v -v Main$ 
+$ exists target/out1
+
+# -- Fail to link without main specified
+> runCli --outpath target/out2 -v -v
+-$ exists target/out2
+
+# -- Fail to link with an incorrect option
+> runCli --outpath target/out3 -v -v --gc fast Main$ 
+-$ exists target/out3
+
+# -- Fail to link with an unspecified option
+> runCli --outpath target/out4 -v -v --unspecified Main$ 
```

Comment:
I think we should not fail with unknown options, instead, we should discard them and print a warning about the ignored option. This way we would not have problems when trying to use an option added in a newer version than we currently use

Reply:
So then we would always treat those as boolean (without an argument)?

Reply:
I think that is what CaseApp does (treat unknown option arguments as program arguments), so I will do the same

Reply:
I was wondering if we should also filter out the value of the argument option, however, I think it would not be needed. It would be simply treated as some, non-existing path and though should not make us any harm